### ISSUE TITLE
Snap Scroll is not Smooth (late reaction)

### ISSUE THREAD

**Issue by aryankarim** [tailwindlabs/tailwindcss#7485]
**What version of Tailwind CSS are you using?**

tailwindcss v.3.0.22

**What build tool (or framework if it abstracts the build tool) are you using?**

postcss v.8.4.6, Next.js v.12.0.7, Webpack v.5

**What version of Node.js are you using?**

Node v.17.3.0

**What browser are you using?**

Chrome Version 97.0.4692.71 (Official Build) (64-bit)

**What operating system are you using?**

Operating System: Manjaro Linux
KDE Plasma Version: 5.23.5
KDE Frameworks Version: 5.90.0
Qt Version: 5.15.2
Kernel Version: 5.15.19-1-MANJARO (64-bit)
Graphics Platform: X11

**Reproduction URL**

All the codes on my chrome browser run like this:
![snap_scroll_bug](https://user-images.githubusercontent.com/58115075/154177521-a19fa04a-8aa2-4281-940a-e7abe95826a3.gif)
But this phenomenon does not occur on my firefox browser.

**Describe your issue**

Snap Scroll is not smooth on Chrome. It has a late reaction when you scroll.

**Comment by adamwathan** [comment]
Hey! Do you get a different result when you use vanilla CSS instead of Tailwind?

**Comment by aryankarim** [comment]
@adamwathan Yes! when I tried this  [codepen link](https://codepen.io/felixvandemaele1/pen/mdBQdpd), it acts the same. Does that mean there is something wrong with my browser's CSS engine? Should I close this issue?

**Comment by NovusOpus** [comment]
I can confirm this also happens with chrome on windows 10/11, but only when you scroll with the mouse wheel(Push the mouse wheel to a side and it'll scroll in the direction you push, Not all Mice support this feature thought) , but it's smooth when you use the arrow keys. But I don't think this is a Tailwind Related bug.

**Comment by adamwathan** [comment]
Got it yeah this sounds like a browser-related thing and not something we could fix by generating different CSS, so I'm going to convert this to a discussion in case anyone has some suggestions for dealing with it (if it can be dealt with at all).
